comp/process/runner/runner.go

```diff
@@ -8,20 +8,24 @@ package runner
 import (
 	"context"
 	"testing"
-	"time"
 
 	"go.uber.org/fx"
 
 	"github.com/DataDog/datadog-agent/comp/core/config"
 	"github.com/DataDog/datadog-agent/comp/core/sysprobeconfig"
 	"github.com/DataDog/datadog-agent/comp/process/submitter"
 	"github.com/DataDog/datadog-agent/comp/process/types"
+	ddconfig "github.com/DataDog/datadog-agent/pkg/config"
+	"github.com/DataDog/datadog-agent/pkg/process/checks"
+	r "github.com/DataDog/datadog-agent/pkg/process/runner"
```

Comment:
TODO: Need a better name for this.

Reply:
I typically just append the higher-level folder name, so in this case it would be `process-runner`

Reply:
Thanks for the suggestion!